components/brave_new_tab_ui/api/brave_news/news.ts

```diff
@@ -79,6 +79,13 @@ class BraveNewsApi {
   }
 
   setPublisherFollowed (publisherId: string, enabled: boolean) {
+    // For now, Direct Sources work differently to Combine Sources - in their
+    // not modified state they are considered enabled.
+    if (this.lastPublishers[publisherId]?.type === PublisherType.DIRECT_SOURCE && !enabled) {
```

Comment:
nit: how about using `isDirectFeed(this.lastPublishers[publisherId]) && !enabled`